modules/ROOT/pages/anypoint-platform-cli.adoc

```diff
@@ -48,48 +56,87 @@ If the git protocol is blocked, you can configure Git to use `https://` instead
 git config --global url."https://".insteadOf git://
 ----
 
+== Authentication
 
-== Reference
+The three ways to configure Anypoint CLI with credentials are username and password, client ID and client secret, or a bearer token.
+At least one method is required.
+
+=== Username and Password
+
+If you don't log in using single sign-on (SSO), you can use your username and password to log into the CLI directly.
```

Comment:
Do you actually "log into the CLI" or do you "access the CLI"?

Reply:
You have to authenticate.